.github/actions/e2e/create-cluster/action.yaml

```diff
@@ -114,8 +114,13 @@ runs:
             roleOnly: true
         withOIDC: true
       addons:
+      # For our scale testing, warm ENIs to help prevent timeouts on binding pods 
```

Comment:
AFAIK, there isn't an assignment of 1 pod:1 ENI. An ENI can support a certain number of Private IPs (this is specified on a per-instance type basis) and a new ENI is only launched once the number of IPs on a given ENI is exhausted.